components/welcome/index.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { TextSampleIcon } from '@/public/icons';
+import SMClickAnimation from '../click-animation';
+
+const texts = [
+  'Once your PR is merged, the token list will update automatically to include your token and your token will be available on the Base Bridge.',
```

Comment:
**Change to:**

Once your PR is merged, the token list will update automatically to include your token on our bridging interface.

Reply:
Done